custom_components/tahoma/coordinator.py

```diff
@@ -59,9 +59,9 @@ def __init__(
         )
 
         self.data = {}
-        self.original_update_interval = update_interval
         self.client = client
         self.devices: Dict[str, Device] = {d.deviceurl: d for d in devices}
+        self.is_stateless = all(len(device.states) == 0 for device in devices)
```

Comment:
For this, I don’t know if it can happen. For instante, just the Tahoma device has 3 states: firmware, Manufacturer, and SupportedManufacturerSettingsCommands

Reply:
I should perhaps add a condition on the type ACTUACTOR. But there is also sensor. What do you thing?

Reply:
Are you sure that firmware and manufacturer is a state? Don't they put that in the attributes?

Reply:
Hum you are right, I read to quickly.

Reply:
I don't own any RTS devices, are we sure that RTS devices (and hubs) don't have anything in the states?

Reply:
I have no clue too :(

Reply:
Let's just merge it, it cannot be worse than before. And based on user complaints / logging, we can investigate further.